app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -18,10 +18,13 @@ package uk.gov.hmrc.tai.config
 
 import play.api.Play._
 import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.tai.model.tai.TaxYear
 import views.html.helper
 
 object ApplicationConfig extends ServicesConfig {
 
+  val statusRange = s"${TaxYear().prev.year}-${TaxYear().prev.year + 1}"
```

Comment:
please use below statement instead of val and TaxYear().prev.year + 1 
`def statusRange = s"${TaxYear().prev.year}-${TaxYear().year}" `

Reply:
Is it possible to write a test case for it ?